htdocs/fourn/commande/list.php

```diff
@@ -737,8 +743,27 @@
 		print $form->selectyesno('validate_invoices', 1, 1);
 		print '</td>';
 		print '</tr>';
+		print '<tr class="fieldrequired" style="display:none">';
+		print '<td>';
+		print $langs->trans('RefSupplier');
+		print '</td>';
+		print '<td>';
+		print '<input name="refsupplier" type="text" id="refsupplier">';
+		print '</td>';
+		print '</tr>';
 		print '</table>';
-
+		?>
+		<script>
+			window.addEventListener('load', function () {
```

Comment:
Js code to check if page is loaded must be done with jquery:
$( document ).ready()